src/components/sharing/ModalPhotosShare.js

```diff
@@ -102,7 +102,6 @@ export class ModalPhotosShare extends Component {
                 <SecuredImageJWT
                   key={"selected_image" + image}
                   height={40}
```

Comment:
I think I would prefer it if the picture is rectangular like here:
https://github.com/LibrePhotos/librephotos-frontend/blob/263e69f063d1b45314dfaba0887a8f86ceabf0f3/src/components/album/ModalAlbumEdit.js#L100-L109

Reply:
This was actually already merged to dev in https://github.com/LibrePhotos/librephotos-frontend/pull/23

Looks like the SecuredImageJWT is making this problematic. I removed the width setting because the thumbnails of landscape photos were squeezed horizontally (although square_thumbnails urls are used). I'm not sure what the SecuredImageJWT is for...

Maybe create a separate issue for making these thumbnails square again?

Reply:
I created two issues https://github.com/LibrePhotos/librephotos/issues/332 and https://github.com/LibrePhotos/librephotos/issues/331 :+1: